variants/XIAO_ESP32C5/pins_arduino.h

```diff
@@ -39,4 +39,7 @@ static const uint8_t D8 = 8;
 static const uint8_t D9 = 9;
 static const uint8_t D10 = 10;
 
+static const uint8_t BAT_Voltage_Read = 6;
+static const uint8_t BAT_Voltage_Read_EN = 26;
```

Comment:
Suggesting a better naming which is used also in other boards.
```suggestion
static const uint8_t BAT_VOLT_PIN = 6;
static const uint8_t BAT_VOLT_PIN_EN = 26;
```